faster_fifo.pyx

```diff
@@ -154,6 +151,21 @@ class Queue:
         return messages
 
     def reallocate_msg_buffer(self, new_size):
-        log.info('Reallocating msg buffer size: %d', new_size)
+        # log.info('Reallocating msg buffer size: %d', new_size)
         self.message_buffer = (ctypes.c_ubyte * new_size)()
         self.message_buffer_memview = memoryview(self.message_buffer)
+
+    def qsize(self):
+        return Q.get_queue_size(<void *>q_addr(self))
+
+    def empty(self):
+        return Q.get_queue_size(<void *>q_addr(self)) == 0
+    
+    def full(self):
```

Comment:
Can you please add the comment here, so it's absolutely clear what this function does:

Return True if the queue is full, False otherwise. If full() returns True it doesn’t guarantee that a subsequent call to get() will not block. Similarly, if full() returns False it doesn’t guarantee that a subsequent call to put() will not block.

This is from https://docs.python.org/3/library/queue.html#queue.Queue.full

Please add similar comment to the empty() function